qiita_client/qiita_client.py

```diff
@@ -28,18 +28,24 @@ class ArtifactInfo(object):
         Qiita's artifact type
     files : list of (str, str)
         The list of (filepath, Qiita's filepath type) that form the artifact
+    archive : dict of {str: str}, optional
+        A dict of features and their values to store. Format: {feature: values}
     """
-    def __init__(self, output_name, artifact_type, files):
+    def __init__(self, output_name, artifact_type, files, archive={}):
```

Comment:
Can you change the default dictionary by a None?

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
didn't see this until now, fixing.

Reply:
BTW is that an anti-pattern for arrays, dicts, or both? Either way changing ...

Reply:
It's an anti-pattern for mutable objects. So arrays, dict, sets, etc, will be an anti-pattern.